docs/blog/chibitronics/v0.1.40.md

```diff
@@ -0,0 +1,107 @@
+# Chibitronics Editor v0.1.40
+
+**Posted on October 8, 2017 by [microsoftsam](https://github.com/microsoftsam)**
+
+We are very excited about all the new features we're unveiling this release. 
+This update brings **Scratch** style rendering of the blocks, a suite of new field editors to enhance the editing experience, a new home screen, a number of simulator fixes and much much more.
+
+
+## Scratch blocks
+
+In this release, we have re-vamped the look of our blocks and aligned with **Scratch**. 
+
+Besides the change in the aesthetic look of the blocks, the main difference here is that the Blockly input (reporter) puzzle piece no longer exists. 
+
+![Scratch blocks before/after comparison](/static/blog/chibitronics/v0.1.40/scratchblockschange.png)
+
+All inputs either have a ROUND or HEXAGONAL shape. 
+If a block has a return type, it will have a shape: 
+- Boolean: HEXAGONAL
+- Number, String, Other: ROUND
+
+![Scratch blocks different shaped blocks](/static/blog/chibitronics/v0.1.40/scratchblocksshapes.png)
+
+
+We've based our open source Blockly fork (pxt-blockly) against [scratch-blocks](https://github.com/LLK/scratch-blocks). 
+
+## New Blocks
+
+We have introduced two new APIs to make the editing experience even simpler. 
+
+**turn PIN ON** block: 
```

Comment:
Just say **turn PIN ON:**. Block is implied and mentioned in the description following.